api/v1alpha1/openapi.yaml

```diff
@@ -1090,33 +1161,88 @@ components:
           format: uuid
         name:
           type: string
-        createdAt:
+        sourceType:
           type: string
-          format: date-time
-        inventory:
-          $ref: '#/components/schemas/Inventory'
-        sourceID:
+          enum: [inventory, rvtools, source]
```

Comment:
```suggestion
          enum: [inventory, rvtools, agent]
          description: >
            Source of the assessment data:
             * `inventory` - Manual inventory upload via JSON
             * `rvtools` - Imported from RVTools Excel file
             * `agent` - Collected by migration planner agent

```
"source" source type might be a bit confusing, what about changing this to "agent" source type?
More than that, maybe we can have more clear types and add information if this "auto" or "manual" upload (like: `invantoryManual`, `rvtoolManual`, `agetAuto`)
And also please document enum values in the property's description field
Any other suggestions here?

Reply:
actually here we don't need rvtools enum value

Reply:
I think we don't need this value at all. Because inventory/agent we disitinguis in agent by onPrem attirbute